js/store.js

```diff
@@ -11,6 +10,91 @@ const store = {
       }
     }
     browser.runtime.onMessage.addListener((m) => store.messageHandler(m));
+
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList;
```

Comment:
@jonathanKingston , for putting constants at the top, do you mean something like this:

```javascript
const store = {
  _websites: null,
  WHITELIST_URL: '/shavar-prod-lists/disconnect-entitylist.json',

  async init() {
    // ...
```
Or outside of the `store` object entirely? Wouldn't we want to keep it inside `store` so as not to "pollute the global namespace"?